ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -126,6 +126,14 @@ final class SandboxServer(
   def this(config: SandboxConfig, materializer: Materializer) =
     this(config, materializer, new Metrics(new MetricRegistry))
 
+  // NOTE(MH): We must do this _before_ we load the first package.
```

Comment:
What happens if we reset the server with the ResetService?

Reply:
To my best understanding, the engine is not touched when the reset service is used. That is at least how I understand the comment
https://github.com/digital-asset/daml/blob/9d955d7039651e670bf58d908f460a96f17456ee/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala#L60

Reply:
Sounds good. I don't particularly understand how resetting affects things; I just know it's easy to get bitten by it.